### PR TITLE
Add extra css

### PR DESCRIPTION
**Pull request by andapop**
- added reset
- moved template from ul, to be able to use `:empty` pseudo element to display the loading message
- added grid for cards and updated css a bit

did not format the files so we can more easily compare the changes, but i have the formatted versions - I would suggest that we use the default prettier settings

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,76 @@
+/* Reset https://www.joshwcomeau.com/css/custom-css-reset/ */
+
+/*
+  1. Use a more-intuitive box-sizing model.
+*/
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
```
Comment (alexnm) [inline src/style.css:9]: we should add this to the starter as well, not just in the final solution. it's actually a great point to discuss with beginners, I remember when I started with html/css this was one of the things that always confused me

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,76 @@
...
+h5,
+h6 {
+  overflow-wrap: break-word;
+}
+/*
+  9. Create a root stacking context - optional
+  this example is useful for a React app that would render in a div with id root
+*/
+#root {
+  isolation: isolate;
```
Comment (alexnm) [inline src/style.css]: I would skip some of these in the starter, to make sure we don't overwhelm people with things they don't know about/don't need for this intro.

We can use it as a reset for the react-starter though, even in a separate file or something

File: src/style.css (hunk 1 above)
Comment (pogadev18) [reply src/style.css:9]: @andapop  feel free to make some of these changes also in this branch `starterImprovement` that has an open PR. We can add pieces of this CSS also for the starter, just like @alexnm  said, and also we can update the HTML with your new changes. :)

File: src/index.html [hunk 3]
```diff
@@ -11,29 +11,41 @@
...
+        data-search
+      />
+      <button class="button" type="submit" id="search-repos-btn">
+        Search
+      </button>
+    </form>
+  </section>
+  <section>
+    <h2 class="u-screen-reader-text">Repositories</h2>
+    <!-- remove content from ul, so we can use :empty to display the loading
```
Comment (pogadev18) [inline src/index.html]: I love this :)

File: src/style.css (hunk 1 above)
Comment (pogadev18) [reply src/style.css:9]: UPDATE: `starterImprovement` branch is now merged so maybe we can create a new branch for the above comment

**Pull request merged**